backend/AppBrowser/Model/User.cs

```diff
@@ -0,0 +1,28 @@
+namespace AppBrowser.Model
+{
+    public class User
+    {
+        public required string Id { get; set; } // from auth
+        public required string FirstName { get; set; }
+        public required string LastName { get; set; }
+        public required string Email { get; set; }
+        public DateTime DateOfBirth { get; set; }
+
+        public DateTime DateOfLicenseObtained { get; set; }
+
+        public required string Location { get; set; }
+
+        public int YearsWithLicense
```

Comment:
Is this correct place for calculating this? Shouldn't we do it in something like `UserService`?